hc/changes.py

```diff
@@ -0,0 +1,2 @@
+def calc(a,b):
+    return a+b
```

Comment:
make this function dynamic so that it does not only add but can also `-`, `x` and `/`